src/main/java/de/tum/in/www1/artemis/domain/quiz/DragItem.java

```diff
@@ -28,10 +32,10 @@
 public class DragItem extends TempIdObject implements QuizQuestionComponent<DragAndDropQuestion> {
```

Comment:
This class has no equals method. Is this simply not needed or do you think we should add one?

Reply:
Neither does the DragAndDropQuestion. It's not really within my scope, so I'd prefer to just leave it like that. If someone needs it at some point, they can implement it.